### DISCUSSION TITLE
Timezone support?

### DISCUSSION THREAD

**Discussion by samuk** [lukevella/rallly#541] [Ideas]
Are there any plans for timezone support?

**Comment by lukevella** [comment]
Rallly already has timezone support. i.e. times are adjusted to each user's timezone.

**Reply by samuk** [reply]
Ah nice, I didn't spot it in the docs. How do you generate it? From IP or Browser settings?

**Reply by samuk** [reply]
OK so playing with it a bit I see how it works now. I don't seem to be able to close this ticket.
